eclipse/sw360
# feat(CRView): Clearing Request Comments enhancements.

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
* Enhancement for Clearing Request Details Page:
* [x]  Move the comment text box field to top.
* [x]  Search box to be displayed on the top right of the comment box UI element (to search the comments).
* [x]  Make both the UI sections (Clearing Request & Comments) remain `expanded`
* [x] Remove the `Comment on clearing decision:` from the CR info .
* [x] Include `Last Updated` date field in the CR info view.

Issue: #879 

### Suggest Reviewer
> 

### How To Test?
* Create a `Clearing Request`, and go to `View/Edit Clearing Request` page. 
* `Last updated on` for clearing request (as shown in point 1 of screenshot)
* Search box for comments (as shown in point 2 of screenshot)
* `Add comment` text box should be displayed on top (as shown in point 3 of screenshot)
* Latest comments should be displayed first (ascending order by comment date)

![Issue879](https://user-images.githubusercontent.com/50870174/84125934-e3920e80-aa5a-11ea-8348-a724d3d543e6.png)

### Checklist
Must:

Signed-off-by: Abdul Kapti <abdul.mannankapti@siemens.com>
